abis/mlibc/signal.h

```diff
@@ -148,11 +206,16 @@ struct sigevent {
 };
 
 struct sigaction {
```

Comment:
So afaict this struct does not actually match the linux kernel's ABI. In the ``sys_sigaction()`` of the linux sysdeps we have this ``struct ksigaction``, which corrects for this mismatch, it would be nice to fix this.

Reply:
Hm, is there a reason why we don't change this directly in the Linux ABI?

Reply:
Not afaik, no.

Reply:
I don't think we can fix that because `sa_flags` is an `int` in the userspace API but `unsigned long` in the kernel. So we probably need to keep that as-is.

Reply:
Ah okay that's annoying.